plugin/hosts/setup_test.go

```diff
@@ -84,3 +84,77 @@ func TestHostsParse(t *testing.T) {
 		}
 	}
 }
+
+func TestHostsInlineParse(t *testing.T) {
+	tests := []struct {
+		inputFileRules      string
+		shouldErr           bool
+		expectedbyAddr      map[string][]string
+		expectedFallthrough bool
+	}{
+		{
+			`hosts inline example.org {
```

Comment:
wait, does this match what the README now says?

Reply:
The example in README is:
```
hosts example.hosts example.org {
    10.0.0.1 example.org
    fallthrough
}
```
where `example.hosts` is the hosts file name. If it is a file path that does not exist (like `inline`) then it will be ignored.

Reply:
ok, cool, I think `highly-likely-to-exist` (used in other tests) is slightly more appropriate here.